cluster/addons/dns/README.md

```diff
@@ -13,6 +13,13 @@ crashes or scheduling changes).  This maps well to DNS, which has a long
 history of clients that, on purpose or on accident, do not respect DNS TTLs
 (see previous remark about Pod IPs changing).
 
+## Where does resolution work?
+
+Cluster DNS resolution standard methods (e.g. [`gethostbyname
+(3)`](http://linux.die.net/man/3/gethostbyname), `dig`) should always work
+within a Pod.  Outside of a Pod, there is no such guarantee: environments will
```

Comment:
I would leave the "Outside of a Pod.." statement out completely, or replace it with "outside of a Pod, the FQDN needs to be used and the cluster nameserver needs to be explicitly specified during DNS lookups"